sdk/openai/azure-ai-openai/src/main/java/com/azure/ai/openai/implementation/OpenAIClientImpl.java

```diff
@@ -1319,6 +1411,9 @@ public Response<BinaryData> getCompletionsWithResponse(String deploymentOrModelN
      *                         ]
      *                     }
      *                 ]
+     *                 refusal (Required): [
+     *                     (recursive schema, see above)
+     *                 ]
```

Comment:
Not entirely sure if this docs are customer visible, but these don't seem to be present in `implementation/NonAzureOpenAIClientImpl.java`

Reply:
We don't export `/implementation` folder to public which is not visible to customers.
File `OpenAIClientImpl` has no manual edited changes exist, so it is purely generated.